api/seeds/development.js

```diff
@@ -7,5 +9,7 @@ exports.seed = async knex => {
   }
 
   // Call specific seeds from here.
+  await truncate.seed(knex);
+  await roles.seed(knex);
```

Comment:
The development seed is opting-in to the "delete everything" and "create roles and activities" seeds.